scripts/teardown.sh

```diff
@@ -55,9 +55,11 @@ gcloud --project="${ISTIO_PROJECT}" compute firewall-rules delete \
 until [[ $(gcloud --project="${ISTIO_PROJECT}" compute firewall-rules list --format "value(name)" \
   --filter "(name:node-http-hc OR name:k8s-fw) AND targetTags.list():gke-${ISTIO_CLUSTER}") == "" ]]; do
   echo "Waiting for firewall rules to delete..."
-  sleep 10
+  sleep 30
 done
 
+sleep 120
```

Comment:
How can we not sleep, but actually get a status of what is going on? Waiting does not work well on the cloud. Time is influenced by multiple factors including GCP region load.

Reply:
The loop above that deletes the firewall rules until they no longer show in a list call was intended to be the retry loop.  The sleep is not a full solution, but rather a way to validate that it is indeed an issue of the FW delete not actually being synchronous.

Reply:
These sleeps were removed.